src/components/ecl-indicator/src/styles/ecl-indicator-ec.scss

```diff
@@ -0,0 +1,3 @@
+@use '@ecl/theme-ec/theme' as theme;
```

Comment:
Do we need this stuff here..? From what i see the scss from ECL is not using any variable from the theme, only custom properties